cms/djangoapps/contentstore/views/tests/test_course_index.py

```diff
@@ -130,7 +132,8 @@ def test_json_responses(self):
         outline_url = reverse_course_url('course_handler', self.course.id)
         chapter = ItemFactory.create(parent_location=self.course.location, category='chapter', display_name="Week 1")
         lesson = ItemFactory.create(parent_location=chapter.location, category='sequential', display_name="Lesson 1")
-        subsection = ItemFactory.create(parent_location=lesson.location, category='vertical', display_name='Subsection 1')
+        subsection = ItemFactory.create(parent_location=lesson.location, category='vertical',
```

Comment:
I think you forgot to reformat this line, and L178 and 180. It's not necessary to reformat lines that you haven't changed in your PR, but if you do want to break them up, please follow the guidelines like you did in L205 and L210